Makefile

```diff
@@ -182,7 +182,7 @@ mimir-build-image/$(UPTODATE): mimir-build-image/*
 # All the boiler plate for building golang follows:
 SUDO := $(shell docker info >/dev/null 2>&1 || echo "sudo -E")
 BUILD_IN_CONTAINER ?= true
-LATEST_BUILD_IMAGE_TAG ?= update-golangci-lint-d48f951b3
+LATEST_BUILD_IMAGE_TAG ?= 5335_d9796228c4f26ab68a87f048f39245de
```

Comment:
Do we need to update image in this PR? We should revert this change and any changes in Dockerfile.

Reply:
no, this is auto-updated by workflow. agreed.

Reply:
I will remove the newline i added in the dockerfile, if there is no change detected, the workflow would not run anymore.